lib/main.js

```diff
@@ -13,7 +13,11 @@ module.exports = function (options) {
   }
 
   if(defaults.ZXingVersion == null) {
-    defaults.ZXingVersion = "-3.1.0"
+    defaults.ZXingVersion = "3.3.0"
```

Comment:
I moved the "-" separator outside the variable, however I wonder now if it was voluntarily put here for some reason in the first place. Do not hesitate to rollback on this if needed.